src/foam/nanos/mrac/ClusterDAO.js

```diff
@@ -97,7 +88,7 @@ foam.CLASS({
       ],
       javaCode: `
       Logger logger = (Logger) x.get("logger");
-      logger.debug(this.getClass().getSimpleName(), "reconfigure", getNSpec().getName());
+      // logger.debug(this.getClass().getSimpleName(), "reconfigure", getNSpec().getName());
```

Comment:
what's will commenting out log statements while we are actively working on this?

Reply:
I removed this line because getNSpec() returns null, which causes an exception.

Reply:
the null is bug. but we can re-add it later if needed.